app/views/unattended/partition_tables_templates/Preseed default cloud-init autoinstall PXELinux.erb

```diff
@@ -0,0 +1,44 @@
+<%#
+kind: PXELinux
+name: Preseed default cloud-init autoinstall PXELinux
+model: ProvisioningTemplate
+oses:
+- Ubuntu 20.04 live-server and hihger
```

Comment:
I'm not sure this is recognized by Foreman's automated matching. AFAIK it only understands the OS name, no versions.